released/packages/coq-mathcomp-analysis/coq-mathcomp-analysis.0.5.2/opam

```diff
@@ -0,0 +1,53 @@
+opam-version: "2.0"
+maintainer: "Reynald Affeldt <reynald.affeldt@aist.go.jp>"
+
+homepage: "https://github.com/math-comp/analysis"
+dev-repo: "git+https://github.com/math-comp/analysis.git"
+bug-reports: "https://github.com/math-comp/analysis/issues"
+license: "CECILL-C"
+
+synopsis: "An analysis library for mathematical components"
+description: """
+This repository contains an experimental library for real analysis for
+the Coq proof-assistant and using the Mathematical Components library."""
+
+build: [make "-j%{jobs}%"]
+install: [make "install"]
+depends: [
+  "coq" { (>= "8.14" & < "8.16~") | (= "dev") }
```

Comment:
If MathComp Analysis is indeed compatible with Coq `dev`, then the upper bound should be changed to `< "8.17~"` at least. Otherwise, the `| (= "dev")` part should be removed.

Reply:
AFAIK, there is no release of coq-elpi and HB for 8.16 but there is a coq-master branch so this indeed doesn't work with Coq 8.16 but does with Coq master.
We could indeed push the bound to `< 8.17~` as this will very likely compile with 8.16 once coq-elpi and HB get a release.